redis/tests/test_async.rs

```diff
@@ -638,3 +638,52 @@ mod pub_sub {
         .unwrap();
     }
 }
+
+#[cfg(feature = "connection-manager")]
+async fn wait_for_server_to_become_ready(client: redis::Client) {
+    let millisecond = std::time::Duration::from_millis(1);
+    let mut retries = 0;
+    loop {
+        match client.get_multiplexed_async_connection().await {
+            Err(err) => {
+                if err.is_connection_refusal() {
+                    tokio::time::sleep(millisecond).await;
+                    retries += 1;
+                    if retries > 100000 {
+                        panic!("Tried to connect too many times, last error: {err}");
+                    }
+                } else {
+                    panic!("Could not connect: {err}");
+                }
+            }
+            Ok(mut con) => {
+                let _: RedisResult<()> = redis::cmd("FLUSHDB").query_async(&mut con).await;
+                break;
+            }
+        }
+    }
+}
+
+#[test]
+#[cfg(feature = "connection-manager")] // TODO - investigate why the test doesn't work with rust-tls
```

Comment:
I'm confused by the TODO here. What's actually failing?

Reply:
My bad, it was fixed the comment should be removed